src/talk/renderer/Settings/DesktopSettingsSection.vue

```diff
@@ -58,6 +59,10 @@ function relaunch() {
 				{{ t('talk_desktop', 'Theme') }}
 			</SettingsSelect>
 
+			<NcCheckboxRadioSwitch :checked.sync="monochromeTrayIcon" type="switch">
```

Comment:
Requires a restart for me to take effect

Reply:
Oops, i forgot

Reply:
Should be good now, you can merge)